tst/teststandard/opers/StructureDescription.tst

```diff
@@ -2,4 +2,8 @@ gap> START_TEST("StructureDescription.tst");
 gap> G := Group([ (6,7,8,9,10), (8,9,10), (1,2)(6,7), (1,2,3,4,5)(6,7,8,9,10) ]);;
 gap> StructureDescription(G);
 "A5 : S5"
+
+## this currently takes a veeeeeery long time (about 20 minutes)
+#gap> StructureDescription(Ree(27));
+#"Ree(27)"
```

Comment:
This had been discussed in #987. Is this example critical to achieve code coverage, or could be easily removed?